tools/install_ubuntu_dependencies.sh

```diff
@@ -63,25 +54,35 @@ function install_ubuntu_common_requirements() {
     qtlocation5-dev \
     qtpositioning5-dev \
     qttools5-dev-tools \
-    libqt5sql5-sqlite \
     libqt5svg5-dev \
-    libqt5charts5-dev \
     libqt5serialbus5-dev  \
     libqt5x11extras5-dev \
-    libqt5opengl5-dev \
-    libreadline-dev
+    libqt5opengl5-dev
 }
 
 # Install extra packages
 function install_extra_packages() {
   echo "Installing extra packages..."
   $SUDO apt-get install -y --no-install-recommends \
-    bzip2 \
-    clinfo \
     casync \
     cmake \
     make \
-    libdw1
+    clinfo \
+    libqt5sql5-sqlite \
+    libqt5charts5-dev \
+    libreadline-dev \
+    libdw1 \
+    libbz2-dev \
```

Comment:
i think we can just delete this. it was for when loggerd would write out compressed logs directly, but the uploader does compression now